scripts/fetch_publisher_jar.sh

```diff
@@ -1,14 +1,31 @@
 #! /bin/bash
 
-# Fetch a version of the IG publisher jar from its remote storage location
-# Get the version from the tools.json file
+# Fetch a version of the IG publisher jar from its official repository
+# Usage ./scripts/fetch_publisher_jar.sh [ a-specific-version ]
 
 set -eo pipefail
 
-EXTRACT_VERSION=$(cat tools.json | jq -r '.publisher.version' | cut -d 'v' -f 2)
-VERSION=${1:-$EXTRACT_VERSION}
+echo "✔ Begin fetching ..."
 
-echo "Fetching IG Publisher version $VERSION"
+VERSION=${1:-'latest'}
 
-JAR_LOCATION="https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=org.hl7.fhir.publisher&a=org.hl7.fhir.publisher.cli&v=$VERSION&e=jar"
-curl -L "$JAR_LOCATION" -o org.hl7.fhir.publisher.jar
+# Parse numeric versioning if given in a 'v\d+.\d+.\d+-SNAPSHOT' format
+if [ $VERSION != 'latest' ]
```

Comment:
If you update `version.json` to store just the semantic version then you won't need this block at all

Reply:
Why don't we keep this parsing though for the wider audience who may be familiar with the old versioning syntax?

Reply:
Hmm ok, shouldn't hurt. That line works for versions in old format or new semantic versioning format